modules/clas/src/main/ui/StudentUi.scala

```diff
@@ -64,7 +64,12 @@ final class StudentUi(helpers: Helpers, clasUi: ClasUi)(using NetDomain):
                     href  := routes.Clas.studentRelease(clas.id, s.user.username),
                     cls   := "button button-empty",
                     title := trans.clas.upgradeFromManaged.txt()
-                  )(trans.clas.release())
+                  )(trans.clas.release()),
+                  a(
+                    href  := routes.Clas.studentMove(clas.id, s.user.username),
+                    cls   := "button button-empty",
+                    title := trans.clas.move.txt()
```

Comment:
superfluous title, it should be `moveToAnotherClass`

Reply:
done